src/module.c

```diff
@@ -266,6 +297,7 @@ NAPI_METHOD(dcn_context_new) {
   dcn_context->napi_event_handler = NULL;
   dcn_context->imap_thread = 0;
   dcn_context->smtp_thread = 0;
+  dcn_context->loop_thread = 0;
```

Comment:
Also, this should be 0, since we don't have any threads running atm. It doesn't _really_ matter though.